deploy/version

```diff
@@ -0,0 +1,2 @@
+# Version of this Deploy configuration (for reference when updating, does not need to be changed)
+WBS_DEPLOY_VERSION=3.0.0
```

Comment:
What is the idea behind this?

Reply:
Good catch. This was from an earlier branch but is now covered by the inclusion of a `package.json` file.